packages/e2e-tests/specs/demo.test.js

```diff
@@ -26,13 +26,15 @@ describe( 'new editor state', () => {
 			},
 		] );
 		await visitAdminPage( 'post-new.php', 'gutenberg-demo' );
+		expect( console ).toHaveWarned();
```

Comment:
We should not expect the console to warn, I think if we address the deprecated call problem, we can remove this expects and we will also fix the end 2 end test.